test/averageRating.spec.js

```diff
@@ -0,0 +1,188 @@
+import { describe, expect, test } from "@jest/globals";
+import { getAverageRating } from "../src/api";
+
+const mockId = 2;
+const mockImdb = "tt2953050";
+const mockImdbRating = 7.8;
+
+//Function to calculate average rating
+function mockMath(mockRating, mockImdbRating) {
+  if (mockRating.data.length >= 5) {
+    let sumOfAll = 0;
+    const mockData = mockRating.data;
+
+    mockData.forEach((int) => {
+      sumOfAll += int.attributes.rating;
+    });
+    const mockAnswer = sumOfAll / mockRating.data.length;
+    return mockAnswer;
+  } else {
+    return mockImdbRating;
+  }
+}
+
+describe("ratingFilterTest()", () => {
+  test("Average rating from user reviews", async () => {
+    const result = await getAverageRating(
+      mockId,
+      mockImdb,
+      mockRating,
+      mockImdbRating
+    );
+    expect(mockRating.data.length).toBeGreaterThan(5);
+    expect(result.rating).toBe(mockMath(mockRating, mockImdbRating));
```

Comment:
Don't use `mockImdbRating` and `mockMath()` here. Instead, just write out the input the values, and the value you expect to get back.